src/coverage-system/coverageservice.ts

```diff
@@ -133,9 +133,15 @@ export class CoverageService {
         if (this.configStore.manualCoverageFilePaths.length) { return; }
 
         const fileNames = this.configStore.coverageFileNames.toString();
+        let baseDir = this.configStore.coverageBaseDir;
+        if (workspace.workspaceFolders) {
+            const workspaceFolders = workspace.workspaceFolders.map((wf) => wf.uri.fsPath);
+            baseDir = `{${workspaceFolders}}/${baseDir}`;
```

Comment:
so the full glob will be as follows if I am understanding this correctly...
`{c/workspacefolderc/coverages, b/workspacefolderb/coverages, a/workspacefoldera/coverages,}`
and then when combined below the blob is:
`{c/workspacefolderc/coverages, b/workspacefolderb/coverages, a/workspacefoldera/coverages,}/{cov.xml, lcov.info}`

If that is correct then I think that works here! as per the other PR you may want to write a few tests to protect this functionality into the future 🤔.

Integration while best might need you to make a new example folder like the remote-node one.. .`example\remote-node`

Reply:
Yes, that is the resulting pattern. I'll add tests and examples

Reply:
I have added unit tests, however not sure what to add to examples. They are used in integration tests, but testing watch changes in-tree is problematic (as described in https://github.com/ryanluker/vscode-coverage-gutters/pull/270#issuecomment-660368276)